en/asgardeo/docs/apis/restapis/actions.yaml

```diff
@@ -14,14 +16,14 @@ paths:
   /actions/types:
     get:
       tags:
-      - Actions Endpoint
+      - Action Types
```

Comment:
By adding this, this API will be shown as a separate category in the side panel. Is it intended ?

Reply:
yes. that's the intention.
Seprated the action type and actions and categorized respective api paths accordingly